packages/fluent/scss/menu/_variables.scss

```diff
@@ -0,0 +1,322 @@
+$kendo-menu-popup-item-icon-margin-start-lg: map-get( $kendo-spacing, 0 ) !default;
+
+/// The end margin of the menu item expand icon.
+/// @group menu
+$kendo-menu-popup-item-icon-margin-end-sm: calc( -1 * ($kendo-icon-size + $kendo-menu-popup-item-padding-x-sm) ) !default;
+$kendo-menu-popup-item-icon-margin-end-md: calc( -1 * ($kendo-icon-size + $kendo-menu-popup-item-padding-x-md) ) !default;
+$kendo-menu-popup-item-icon-margin-end-lg: calc( -1 * ($kendo-icon-size + $kendo-menu-popup-item-padding-x-lg) ) !default;
+
+/// The base shadow of focused menu item in popup.
+/// @group menu
+$kendo-menu-popup-focus-outline-offset: $kendo-menu-item-focus-outline-offset !default;
+$kendo-menu-popup-focus-outline-width: $kendo-menu-item-focus-outline-width !default;
+$kendo-menu-popup-focus-outline-style: $kendo-menu-item-focus-outline-style !default;
+
+/// The spacing between the menu items in popup.
+/// @group menu
+$kendo-menu-popup-item-spacing: map-get( $kendo-spacing, 0 ) !default;
+
+/// Sizes map for the menu.
+/// @group menu
+$kendo-menu-sizes: (
+    sm: (
+        group: (
+            font-size: $kendo-menu-popup-font-size-sm,
+            line-height: $kendo-menu-popup-line-height-sm,
+        ),
+        link: (
+            padding-x: $kendo-menu-popup-item-padding-x-sm,
+            padding-y: $kendo-menu-popup-item-padding-y-sm,
+            padding-inline-end: $kendo-menu-popup-item-padding-end-sm,
+        ),
+        arrow: (
+            margin-inline-start: $kendo-menu-popup-item-icon-margin-start-sm,
+            margin-inline-end: $kendo-menu-popup-item-icon-margin-end-sm,
+        )
+    ),
+    md: (
+        group: (
+            font-size: $kendo-menu-popup-font-size-md,
+            line-height: $kendo-menu-popup-line-height-md,
+        ),
+        link: (
+            padding-x: $kendo-menu-popup-item-padding-x-md,
+            padding-y: $kendo-menu-popup-item-padding-y-md,
+            padding-inline-end: $kendo-menu-popup-item-padding-end-md,
+        ),
+        arrow: (
+            margin-inline-start: $kendo-menu-popup-item-icon-margin-start-md,
+            margin-inline-end: $kendo-menu-popup-item-icon-margin-end-md,
+        )
+    ),
+    lg: (
+        group: (
+            font-size: $kendo-menu-popup-font-size-lg,
+            line-height: $kendo-menu-popup-line-height-lg,
+        ),
+        link: (
+            padding-x: $kendo-menu-popup-item-padding-x-lg,
+            padding-y: $kendo-menu-popup-item-padding-y-lg,
+            padding-inline-end: $kendo-menu-popup-item-padding-end-lg,
+        ),
+        arrow: (
+            margin-inline-start: $kendo-menu-popup-item-icon-margin-start-lg,
+            margin-inline-end: $kendo-menu-popup-item-icon-margin-end-lg,
+        )
+    )
+) !default;
+
+
+// Menu item in popup
+
+/// The text color of the menu item in popup.
+/// @group menu
+$kendo-menu-popup-item-text: $kendo-list-item-text !default;
```

Comment:
add all of these variables as CSS variables: 
var( --kendo-list-item-text, #{$kendo-list-item-text} )

In this way if the list css variables are used for customization, it would apply on the menu as well.